src/ImageSearchModal.jsx

```diff
@@ -170,48 +157,58 @@ export class ImageSearchModal extends React.Component {
                                 })
                                 }
                             </Select.Select>
-                        </div>
-                    </div>
-                </InputGroup>
+                        </FormGroup>
+                    </Flex>
+                </Form>
 
-                {this.state.searchInProgress && <div id='search-image-dialog-waiting' className='spinner' />}
+                {<Bullseye><Spinner id='search-image-dialog-waiting' /></Bullseye>}
```

Comment:
There is spinner when I open "get new image". The spinner should be there only when it is searching for something. It is also there when  I get some results or even when there are no results.

It needs that `this.state.searchInProgress` check

Reply:
Done